wdio.conf.js

```diff
@@ -59,7 +59,7 @@ exports.config = {
   framework: 'jasmine',
   jasmineOpts: { // used to be jasmineNodeOpts; when jasmine.DEFAULT_TIMEOUT_INTERVAL is 10000 this isn't being taken into account
     ui: 'bdd',
-    defaultTimeoutInterval: 60000,
+    defaultTimeoutInterval: 80000,
```

Comment:
We're already waiting for a full minute. With this change we wait for 80 seconds? Why the additional 20 seconds? If it's a change for a small set of tests / suite / something, is there some way to set the timeout specifically for those tests? The AQA plan is already running super long; adding an additional 20 seconds for every test that's going to time out will just add even more time to it. Can we avoid that in the general case?

Reply:
This was added only because of add new complaint screen loads slow in dev but not in other environment. (checked manually)

Reply:
Also not really okay with this either, but, again, am going to trust you